lightkurve/lightcurve.py

```diff
@@ -1609,6 +1609,7 @@ def _make_aperture_extension(hdu_list, aperture_mask):
         hdu = super(TessLightCurve, self).to_fits(path=None,
                                                     overwrite=overwrite,
                                                     **extra_data)
+        hdu[1].columns.change_name('SAP_QUALITY', 'QUALITY')
```

Comment:
Maybe add a comment here to make it explicit that we do this because the TESS file format is subtly different in the name of this column.